### DISCUSSION TITLE
Fork For Raspberry pi

### DISCUSSION THREAD

**Discussion by dnalexxio** [jomjol/AI-on-the-edge-device#2599] [Ideas] [answered]
Hello, i would like to create a fork for raspberry pi of it, to test the functionalities without taking the whole esp32 hardware. 
can you please help letting me understand which part can i extract?

**Comment by jomjol** [comment, marked as answer]
The structure of the code is strongly interlinked with the hardware. It only makes sense to transfer the neural network part itself.
The core of it is in /components/jomjol_tfliteclass. But it needs a lot of supporting libraries around.
